### PR TITLE
Add `timeago` for human readable dates, switch to ESM

### PR DESCRIPTION
**Pull request by rbuckton**
Pushing up some local QOL changes prior to updating to use new dashboard.

### CODE REVIEW

File: bin/package.json [hunk 1]
```diff
@@ -0,0 +1 @@
+{ "type": "commonjs" }
```
Comment (jakebailey) [inline bin/package.json:1]: Any reason not just use ESM here? I could have sworn ESM worked in these extensionless files...

File: src/prompt.ts [hunk 2]
```diff
@@ -1,6 +1,6 @@
 import * as readline from "readline";
 import chalk from "chalk";
-import stripAnsi from "strip-ansi";
+import stripAnsi = require("strip-ansi");
```
Comment (jakebailey) [inline src/prompt.ts:3]: Does this compile? This is CJS syntax in ESM but maybe it works?

**Review by jakebailey — APPROVED** [review]

File: src/prompt.ts (hunk 2 above)
Comment (rbuckton) [reply src/prompt.ts:3]: It's a feature of `module:nodenext`, @weswigham added support for transforming this to use `createRequire`.

File: bin/package.json (hunk 1 above)
Comment (rbuckton) [reply bin/package.json:1]: It's been awhile since I wrote this bit, but it wasn't working at the time.

**Pull request merged**